### PR TITLE
Vanish specific minifier JSMin from ezjscore as it's now possible to use whatever js/css minifier

### PR DESCRIPTION
**Pull request by jdespatis**
As discussed with André on #18165 and #18431, it's now possible to choose whatever js/css minifier, as a result:
1/ ezPublish should not bundle anymore JSMin, which is one minifier among others (why include JSMin and not the other ones? so ezp should bundle none of them)
2/ if a developer wants to include in his/her extension JSMin minifier, he/she gets an error on regenerate autoload because of the already included JSMin
3/ Further more the developer may want to include another fork of JSMin (same name but different code)
4/ JSMin can minifier with no risk code that has been verified by JSLint, which is not the case indeed. Think of jquery, it's already minified, so is really not JSLint friendly, so big big risk to run JSMin on it. The fact is using JSMin is very dangerous in ezpublish, so no one should use packLevel = 3 for javascript. Much better to use another js minifier.

### CODE REVIEW

File: autoloads/ezjscpackertemplatefunctions.php [hunk 1]
```diff
@@ -57,7 +57,7 @@
  *  0 = off
  *  1 = merge files
  *  2 = 1 + remove whitespace
- *  3 = 2 + remove more whitespace  (jsmin is used for scripts)
+ *  3 = 2 + remove more whitespace
```
Comment (andrerom) [inline autoloads/ezjscpackertemplatefunctions.php]: Might want to change the text to say "3 = 2 (deprecated)"

**Comment by jdespatis** [comment]
Here it is @andrerom

**Comment by andrerom** [comment]
And then it was merged, thanks :)

**Comment by bdunogier** [comment]
woot, well done guys.

**Pull request merged**